### PR TITLE
Newboard assurance

### PR DESCRIPTION
**Pull request by jon-shanks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This page will give you an overview of HOD DSP, the services it entails and the ways to get access to them. This is a general overview. Developers should check out the [Developer Docs](https://github.com/UKHomeOffice/hosting-platform/tree/master/developer-docs).
 
+## Service Status
+
+For an overview of the platform services we provide and their current status i.e. Alpha, Beta, Live; please refer to the [platform service status page](docs/platform_service_status.md). This is to give consumers of the platform a single reference point on what services we are providing and where each service is in the lifecycle. For definitions on each phase of the lifecycle and the criteria for production-ready; please refer [here](docs/service_lifecycle.md).
```
Comment (rickwilliams391) [inline README.md]: a single reference point on **the** services..........

File: docs/platform_service_status.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Platform Service Status
+
+This provides a status of our overall services and where in the lifecycle they are in their evolution.
+There are service lifecycle definitions [here](../docs/service_lifecycles.md). This is to help projects understand where the gaps might be on some of the services they might want to consume and therefore assess the risk related to them.
```
Comment (rickwilliams391) [inline docs/platform_service_status.md]: Service lifecycle definitions can be found here

File: docs/service_lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Service Lifecycle
+
+This defines the current service lifecycle stages, this isn't specific to the hosting platform
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: full stop after stages. These are generic service definitions which are intended to clearly communicate the maturity of each service.

File: docs/service_lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+the direction and the intention of the service. It is there to explore ways of achieving and meeting
+the user needs in the right way. 
+
+There is more information on this [here](https://www.dta.gov.au/standard/service-design-and-delivery-process/) and
+[here](http://ausdto.github.io/service-handbook/alpha/1-introduction/1-1-what.html)
+
+
+## Beta
+
+This is the phase where software is more feature complete, it is aimed at being functional and meeting the definition
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: feature complete. It is.........

File: docs/service_lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+of a Minimal Viable Product, (MVP). The service is accessible and secured and meets a lot of the production requirements and needs,
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: ....production requirements and needs. However, there will be an assessment of the service through closed user groups to evaluate that it meets the needs as expected.

File: docs/service_lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+however, there will be a assessment of the service through closed user groups; to evaluate that it meets the needs as expected.
+SLA's and SLO's may differ from the user expectation of a live service and the availability expectation may not be to the standards
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: SLA's, SLO's and service availability may not meet the user expectation of a live service.

File: docs/service_lifecycle.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+people would want of a live service. 
+
+## Production
+
+Once the product's have gone through the beta process, (which is to identify that the MVP is viable), the service will transition
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: ....the service will transition to production. In production, there will be more rigour around......

File: docs/service_lifecycle.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+into more of a production role. This means that it will have more rigour around the service, defined SLA's and SLO's to meet user
+dependencies and expectations around the service. 
+
+## What is the beta / production criteria.
```
Comment (rickwilliams391) [inline docs/service_lifecycle.md]: What is production ready?

**Comment by rickwilliams391** [comment]
Added comments to wording

**Comment by rickwilliams391** [comment]
Looks good.

File: README.md [hunk 9]
```diff
@@ -2,6 +2,10 @@
 
 This page will give you an overview of HOD DSP, the services it entails and the ways to get access to them. This is a general overview. Developers should check out the [Developer Docs](https://github.com/UKHomeOffice/hosting-platform/tree/master/developer-docs).
 
+## Service Status
+
+For an overview of the platform services we provide and their current status i.e. Alpha, Beta, Live; please refer to the [platform service status page](docs/platform_service_status.md). This is to give consumers of the platform a single reference point on the services we are providing and where each service is in the lifecycle. For definitions on each phase of the lifecycle and the criteria for production-ready; please refer [here](docs/service_lifecycle.md).
+
```
Comment (Joseph-Irving) [inline README.md]: That's not an independent clause so it should be a comma not a semi colon

File: docs/platform_service_status.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
+# Platform Service Status
+
+This provides a status of our overall services and where in the lifecycle they are in their evolution.
+Service lifecycle definitions can be found [here](../docs/service_lifecycles.md). This is to help projects understand where the gaps might be on some of the services they might want to consume and therefore assess the risk related to them.
```
Comment (Joseph-Irving) [inline docs/platform_service_status.md]: "This provides the status of all of our services in regards to where they are in the development lifecycle" maybe nicer

File: docs/service_lifecycle.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# Service Lifecycle
+
+This defines the current service lifecycle stages. This isn't specific to the hosting platform
+and will no doubt move to somewhere more centric and referenced. However, this is a starting 
+point to clearly communication the phases of our evolution. 
+
```
Comment (Joseph-Irving) [inline docs/service_lifecycle.md]: communicate

File: docs/service_lifecycle.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+This is the phase where software is more feature complete. It is aimed at being functional and meeting the definition
+of a Minimal Viable Product, (MVP). The service is accessible and secured and meets a lot of the production requirements and needs.
+However, there will be an assessment of the service through closed user groups, to evaluate that it meets the user needs.
+SLA's, SLO's and availability may not meet the user expectation of a live service.
+
+## Live
+
+Once the product's have gone through the beta process, (which is to identify that the MVP is viable), the service will transition to live.
+In live, we are happy that the service has undergone enough evaluation to meet user expectations as well as being able to commit confidentally to SLA's and SLO's.
```
Comment (Joseph-Irving) [inline docs/service_lifecycle.md]: products not product's

**Comment by jon-shanks** [comment]
Updated grammar and rephrased, didn't go with development lifecycle but worded in a similar way.

File: README.md [hunk 13]
```diff
@@ -2,6 +2,10 @@
 
 This page will give you an overview of HOD DSP, the services it entails and the ways to get access to them. This is a general overview. Developers should check out the [Developer Docs](https://github.com/UKHomeOffice/hosting-platform/tree/master/developer-docs).
 
+## Service Status
+
+For an overview of the platform services we provide and their current status i.e. Alpha, Beta, Live, please refer to the [platform service status page](docs/platform_service_status.md). This is to give consumers of the platform a single reference point on the services we are providing and where each service is in the lifecycle. For definitions on each phase of the lifecycle and the criteria for production-ready; please refer [here](docs/service_lifecycle.md).
+
```
Comment (Joseph-Irving) [inline README.md]: I meant the second semi colon, using a semi colon after "Alpha, Beta, Live;" is totally correct

File: docs/service_lifecycle.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+- Highly available (Multiple Instances)
+- Backups of all data
+- Validation of backup, (not 0 file size)
+- Restores of data tested
+- CI Release process
+- Continuous deployment (if needed) to prod, (done via CI)
+- Is it independent of itself i.e. it doesn't rely on itself
+- Tests defined and ran as part of CI process
+- Monitoring of product
+- Does it have an ATO?
```
Comment (jaykeshur) [inline docs/service_lifecycle.md]: Change to Authority to Operate (acronyms are hard to remember)

**Pull request merged**